tests/integration/docker/mtu.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# MTU testing using Swarm
+
+SRC="${BATS_TEST_DIRNAME}/../../lib/"
+# number of replicas to launch
+number_of_replicas=1
+# number of attempts
+number_of_attempts=5
+
+#This function will verify that swarm is not running or it finishes
+function clean_swarm_status() {
```

Comment:
What about moving this function to `tests/lib/test-common.bash` so we do not have a duplicate function here and on swarm.bats?